### PR TITLE
first html layout

### PR DESCRIPTION
**Pull request by marzieh67**

### CODE REVIEW

**Comment by MrTim** [comment]
Hi @marzieh67  could you maybe add a short description what this layout is for? Because it's hard to guess based only on the code :)

File: index.html [hunk 1]
```diff
@@ -1 +1,57 @@
-<!DOCTYPE html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+
```
Comment (MrTim) [inline index.html:5]: You don't need these empty lines

File: index.html [hunk 2]
```diff
@@ -1 +1,57 @@
...
+    <!--CSS & js LINK-->
+
+    <title>Weather App</title>
+
+</head>
+<body>
+<section>
+    <div class="search contain">
```
Comment (MrTim) [inline index.html:13]: you're using classes but there is no CSS file linked to this file. Please next time create all styles and link them here.

File: index.html [hunk 3]
```diff
@@ -1 +1,57 @@
...
+        <h1>Weather App</h1>
+        <form>
+           City: <input type="text" placeholder="name of city">
+           <button type="submit">Search</button>
+        </form>
+
```
Comment (MrTim) [inline index.html:19]: this empty line is not needed as well

File: index.html [hunk 4]
```diff
@@ -1 +1,57 @@
...
+    <div class="weekly prediction">
+        <div class = "1st day prediction"></div>
+        <div class = "2nd day prediction"></div>
+        <div class = "3rd day prediction"></div>
+        <div class = "4th day prediction"></div>
+        <div class = "5th day prediction"></div>
+        <div class = "6th day prediction"></div>
+        <div class = "7th day prediction"></div>
+    </div>
+
```
Comment (MrTim) [inline index.html:37]: empty lines)

File: index.html [hunk 5]
```diff
@@ -1 +1,57 @@
...
+
+    
+</section>
+
+<!--Should be done:
```
Comment (MrTim) [inline index.html:42]: This section and empty lines as separators you don't need. When you get to the point when it's needed then you will add this section.

**Review by MrTim — CHANGES_REQUESTED** [review]
Good start. Left some comments, please clean up them and we'll check again.

File: index.html (hunk 5 above)
Comment (marzieh67) [reply index.html:33]: @MrTim Thank you Tim , I deleted the extra lines.

**Review by MrTim — APPROVED** [review]
Cool! Now you're good to go in merging this pr to master.

**Pull request merged**